package.json

```
@@ -35,11 +37,12 @@
   },
   "scripts": {
     "docker:build": "yarn workspace @nordeck/matrix-neoboard-widget docker:build",
-    "start": "yarn workspace @nordeck/matrix-neoboard-widget start",
     "dev": "yarn workspace @nordeck/matrix-neoboard-widget dev",
+    "dev:https": "yarn workspace @nordeck/matrix-neoboard-widget dev:https",
     "build": "yarn workspaces run build",
-    "test": "yarn workspaces run test",
-    "test:all": "yarn workspaces run test:all",
+    "test": "vitest watch",
```

This doesn't need to use `yarn workspace` because vitest does it's own workspace thing, right?

```suggestion
    "test": "vitest",
```

---

yeah vitest actually _requires_ their own thing for workspaces. Somehow they dont support workspaces otherwise and it lead to errors when I did it the first time around.